### ISSUE TITLE
Fails to run on Windows

### ISSUE THREAD

**Issue by madduck** [ProCern/asyncinotify#13]
```
Traceback (most recent call last):
  File "C:\Users\martin\tptools\webserver.py", line 6, in <module>
    from asyncinotify import Inotify, Mask
  File "C:\Users\martin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCa
che\local-packages\Python312\site-packages\asyncinotify\__init__.py", line 23, in <module>
    from . import _ffi
  File "C:\Users\martin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCa
che\local-packages\Python312\site-packages\asyncinotify\_ffi.py", line 34, in <module>
    libc = ctypes.CDLL(_libcname, use_errno=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.2288.0_x64__qbz5n2kfra8p0\
Lib\ctypes\__init__.py", line 369, in __init__
    if '/' in name or '\\' in name:
       ^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable
```

This seems because `_libcname` is `None`, which [has been anticipated](https://github.com/ProCern/asyncinotify/blob/master/src/asyncinotify/_ffi.py#L31), but it doesn't work on Windows.

**Comment by wyrmiyu** [comment]
I am not sure if that particular error can be fixed, but it really does not matter. This library depends on [`inotify(7)`](https://man7.org/linux/man-pages/man7/inotify.7.html) which is a Linux kernel [subsystem](https://en.wikipedia.org/wiki/Inotify). Thus, it cannot work on Windows directly.

Seems that Microsoft has implemented [`inotify` support for WSL1 ](https://devblogs.microsoft.com/commandline/wsl-adds-inotify-filesystem-change-notification-support/), but [for WSL2 there is a feature request from 2019](https://github.com/microsoft/WSL/issues/4739) that to this date has not been addressed.

**Comment by madduck** [comment]
> I am not sure if that particular error can be fixed, but it really does not matter. This library depends on [`inotify(7)`](https://man7.org/linux/man-pages/man7/inotify.7.html) which is a Linux kernel [subsystem](https://en.wikipedia.org/wiki/Inotify). Thus, it cannot work on Windows directly.

This does not follow from the docs, and should probably be added there. Even better would be a runtime check for Windows, and an approriate error message.

**Comment by TaylorRichberger** [comment]
It does follow from the docs.  This is "An async python inotify package".  A web search for "inotify" makes it abundantly clear that this is a Linux-only API.  If you know what "inotify" means, you know where you can use this.  If you don't know what "inotify" means, why would you pull a "python inotify package"?

There is no reason to have a runtime check, any more than you'd add one for a Python systemd library.

**Comment by madduck** [comment]
@TaylorRichberger Yes, I know what inotify is. I am suggesting you make the error nicer for those who don't and/or just make it explicit to users in the docs that this won't run on Windows.

Yes, there is a case to be made about nobody on Windows possibly ever finding an inotify library.

There is also a case to be made about just being explicit rather than assuming others know.
